source/PlayerInfo.cpp

```diff
@@ -1443,7 +1445,8 @@ bool PlayerInfo::TakeOff(UI *ui)
 			// Compute the total value for each type of excess outfit.
 			if(!outfit.second)
 				continue;
-			int64_t cost = depreciation.Value(outfit.first, day, outfit.second);
+			int64_t cost = depreciation.Value(outfit.first, day, outfit.second, 
+				planet->Outfitter().GetSold(outfit.first)->GetCost());
```

Comment:
What if the planet doesn't have an outfitter that sells the outfit? I think it will crash since `GetSold` will return `nullptr`.

Reply:
it's even worse than that I had forgotten to put the arguments in the right order
I did this before I did the GetCost(outfit) function for outfitsale, will change